keyboards/id67/rules.mk

```diff
@@ -8,15 +8,16 @@ BOOTLOADER = atmel-dfu
 #   change yes to no to disable
 #
 BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
-MOUSEKEY_ENABLE = yes       # Mouse keys
+MOUSEKEY_ENABLE = no        # Mouse key
```

Comment:
```suggestion
MOUSEKEY_ENABLE = no        # Mouse keys
```

Reply:
Are you disabling this feature because the firmware does not fit into flash? Some existing configurator and VIA users may not like this. Maybe it would be better to disable some RGB effects instead.

Reply:
I disabled it by mistake, with this option enabled it still fits.